pennylane/templates/layers/particle_conserving_u1.py

```diff
@@ -351,4 +360,111 @@ def shape(n_layers, n_wires):
             raise ValueError(
                 f"The number of qubits must be greater than one; got 'n_wires' = {n_wires}"
             )
+
         return n_layers, n_wires - 1, 2
+
+
+def _particle_conserving_u1_resources(n_layers: int, num_wires: int):
+    # number of pairs of even-indexed of wires
+    num_nm_wires = (
+        math.floor(num_wires / 2) if num_wires % 2 == 0 else math.ceil((num_wires - 1) / 2)
+    )
+    # number of odd-indexed pairs of wires
+    num_nm_wires += math.floor((num_wires - 1) / 2)
```

Comment:
See `ParticleConservingU2` PR

```suggestion
    num_nm_wires = num_wires-1
```